jsons/Nations.json

```diff
@@ -1438,16 +1438,34 @@
+            "Instantly gain [250] [GoldenAgePoints] <upon adopting [Piety Complete]> <hidden from users>",
+            "Instantly gain [250] [GoldenAgePoints] <upon adopting [Aesthetics Complete]> <hidden from users>",
+            "Instantly gain [250] [GoldenAgePoints] <upon adopting [Patronage Complete]> <hidden from users>",
+            "Instantly gain [250] [GoldenAgePoints] <upon adopting [Exploration Complete]> <hidden from users>",
+            "Instantly gain [250] [GoldenAgePoints] <upon adopting [Commerce Complete]> <hidden from users>",
+            "Instantly gain [250] [GoldenAgePoints] <upon adopting [Rationalism Complete]> <hidden from users>",
+            "Empire enters a [5]-turn Golden Age <during a Golden Age> <upon adopting [Tradition Complete]> <hidden from users>",
+            "Empire enters a [5]-turn Golden Age <during a Golden Age> <upon adopting [Liberty Complete]> <hidden from users>",
+            "Empire enters a [5]-turn Golden Age <during a Golden Age> <upon adopting [Honor Complete]> <hidden from users>",
+            "Empire enters a [5]-turn Golden Age <during a Golden Age> <upon adopting [Piety Complete]> <hidden from users>",
+            "Empire enters a [5]-turn Golden Age <during a Golden Age> <upon adopting [Aesthetics Complete]> <hidden from users>",
+            "Empire enters a [5]-turn Golden Age <during a Golden Age> <upon adopting [Patronage Complete]> <hidden from users>",
+            "Empire enters a [5]-turn Golden Age <during a Golden Age> <upon adopting [Exploration Complete]> <hidden from users>",
+            "Empire enters a [5]-turn Golden Age <during a Golden Age> <upon adopting [Commerce Complete]> <hidden from users>",
+            "Empire enters a [5]-turn Golden Age <during a Golden Age> <upon adopting [Rationalism Complete]> <hidden from users>",
+            "Instantly gain [-250] [GoldenAgePoints] <during a Golden Age> <upon adopting [Tradition Complete]> <hidden from users>",
+            "Instantly gain [-250] [GoldenAgePoints] <during a Golden Age> <upon adopting [Liberty Complete]> <hidden from users>",
+            "Instantly gain [-250] [GoldenAgePoints] <during a Golden Age> <upon adopting [Honor Complete]> <hidden from users>",
+            "Instantly gain [-250] [GoldenAgePoints] <during a Golden Age> <upon adopting [Piety Complete]> <hidden from users>",
+            "Instantly gain [-250] [GoldenAgePoints] <during a Golden Age> <upon adopting [Aesthetics Complete]> <hidden from users>",
+            "Instantly gain [-250] [GoldenAgePoints] <during a Golden Age> <upon adopting [Patronage Complete]> <hidden from users>",
+            "Instantly gain [-250] [GoldenAgePoints] <during a Golden Age> <upon adopting [Exploration Complete]> <hidden from users>",
+            "Instantly gain [-250] [GoldenAgePoints] <during a Golden Age> <upon adopting [Commerce Complete]> <hidden from users>"
+            //maybe need better solution for condition
```

Comment:
PolicyFilter can work with unique, so you can give all finishers a "finisher" unique and this becomes one line